cluster-scope/overlays/prod/osc/osc-cl1/groups/sudoers.yaml

```diff
@@ -1,11 +1,10 @@
 apiVersion: user.openshift.io/v1
 kind: Group
 metadata:
-  name: cluster-admins
+  name: sudoers
```

Comment:
by removing us as cluster-admins, what sort of permissions do we require to be able to impersonate this group from the console? are these already available?